### PR TITLE
Change beta.protonmail.com to mail.protonmail.com

### PR DESCRIPTION
**Pull request by fishstik**
ProtonMail beta has gone stable. Update dynamic theme config accordingly.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7289,6 +7277,12 @@ select.field {
     background: var(--darkreader-neutral-background) !important;
     color: var(--darkreader-neutral-text) !important;
 }
+.label-stack-item {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7280]: Are these fixes still needed? I know they've had some changes in the layout since that fix where made.

Comment (fishstik) [reply src/config/dynamic-theme-fixes.config:7280]: Yes, without them, labels show like so:
image

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:7280]: Got it 👍🏽

**Review by Gusted — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**